Kubernetes/windows/start-kubelet.ps1

```diff
@@ -200,26 +200,27 @@ if (-not $podCidrDiscovered)
 $podGW = Get-PodGateway $podCIDR
 ipmo C:\k\hns.psm1
 
-# At this point, no SDN network should exists 
-
-if((Get-HnsNetwork | ? Type -EQ "L2Bridge") -eq $null )
+# Create a L2Bridge to trigger a vSwitch creation. Do this only once
+if(!(Get-HnsNetwork | ? Name -EQ "External"))
 {
-    $hnsNetwork = New-HNSNetwork -Type $NetworkMode -AddressPrefix $podCIDR -Gateway $podGW -Name $NetworkName.ToLower() -Verbose
+    New-HNSNetwork -Type $NetworkMode -AddressPrefix "192.168.255.0/30" -Gateway "192.168.255.1" -Name "External" -Verbose
```

Comment:
Can we add parameters or variables for this? I think it would be best if we didn't make any hardcodings. Otherwise it might overlap with existing subnets, and people may not know what is causing interference.